integ-test/src/test/java/org/opensearch/sql/calcite/remote/CalciteMultiValueStatsIT.java

```diff
@@ -0,0 +1,1920 @@
+      Assertions.assertFalse(((String) value).trim().isEmpty(), "Name should not be empty");
+    }
+  }
+
+  @Test
+  public void testListFunctionWithNumericField() throws IOException {
+    JSONObject response =
+        executeQuery(
+            String.format(
+                "source=%s | where account_number < 5 | stats list(account_number) as nums",
+                TEST_INDEX_ACCOUNT));
+
+    verifySchema(response, schema("nums", null, "array"));
+
+    JSONArray dataRows = response.getJSONArray("datarows");
+    Assertions.assertEquals(1, dataRows.length(), "Should return exactly one aggregation row");
+    System.out.printf("result rows: %s%n", dataRows);
+
+    // Verify the nums array contains string representations of numbers
+    JSONArray nums = dataRows.getJSONArray(0).getJSONArray(0);
+    System.out.printf("result: %s%n", nums);
+    Assertions.assertTrue(nums.length() > 0, "nums array should not be empty");
+
+    for (int i = 0; i < nums.length(); i++) {
+      Object value = nums.get(i);
+      Assertions.assertTrue(
+          value instanceof String, "All account_number values should be strings: " + value);
+
+      String numStr = (String) value;
+      Assertions.assertTrue(
+          numStr.matches("\\d+"), "Account number should be numeric string: " + numStr);
+
+      int accountNum = Integer.parseInt(numStr);
+      Assertions.assertTrue(
+          accountNum >= 0 && accountNum < 5,
+          "Account number should be in expected range: " + accountNum);
+    }
+  }
+
+  @Test
+  public void testValuesFunctionWithStringField() throws IOException {
+    JSONObject response =
+        executeQuery(
+            String.format(
+                "source=%s | where account_number < 5 | stats values(gender) as unique_genders",
+                TEST_INDEX_ACCOUNT));
+
+    verifySchema(response, schema("unique_genders", null, "array"));
+
+    JSONArray dataRows = response.getJSONArray("datarows");
+    Assertions.assertEquals(1, dataRows.length(), "Should return exactly one aggregation row");
+
+    // Verify the unique_genders array contains sorted unique values
+    JSONArray genders = dataRows.getJSONArray(0).getJSONArray(0);
+    Assertions.assertTrue(genders.length() > 0, "genders array should not be empty");
+
+    // Verify lexicographic sorting and uniqueness
+    String previousGender = null;
+    for (int i = 0; i < genders.length(); i++) {
+      Object value = genders.get(i);
+      Assertions.assertTrue(
+          value instanceof String, "All gender values should be strings: " + value);
+
+      String currentGender = (String) value;
+      if (previousGender != null) {
+        Assertions.assertTrue(
+            currentGender.compareTo(previousGender) > 0,
+            "Values should be in lexicographic order: " + previousGender + " vs " + currentGender);
+      }
+      previousGender = currentGender;
+    }
+  }
+
+  @Test
+  public void testValuesWithPercentileApproxUdaf() throws IOException {
```

Comment:
np: I think we can simplify this IT

1. IMO, test like this is not necessary
2. To test list/values with different data type, we can consider parameterized test to deduplicate code
3. Can the test without pushdown covered by `CalciteNoPushdownIT` as checklist mentioned?